channels/co.m3u

```diff
@@ -59,10 +63,6 @@ https://telecaribe-hd.gcdn.anvato.net/hls/live/telecaribehd/2196k/index.m3u8
 https://telecaribe-hd.gcdn.anvato.net/hls/live/telecaribehd/3296k/index.m3u8
 #EXTINF:-1 tvg-id="Telecaribe.co" tvg-name="Telecaribe" tvg-country="CO" tvg-language="Spanish" tvg-logo="https://i.imgur.com/i5wyEor.png" group-title="",Telecaribe
 https://telecaribe-hd.gcdn.anvato.net/hls/live/telecaribehd/496k/index.m3u8
-#EXTINF:-1 tvg-id="TeleCaribeDeportes2.co" tvg-name="TeleCaribe Deportes 2" tvg-country="CO" tvg-language="" tvg-logo="" group-title="",TeleCaribe Deportes 2
```

Comment:
I don't see a reason to remove this channel. It's not like the others

Reply:
It gets to a while back. Reason for this is that it shows the same programme over and over again. (just try by looking at the Channel twice, and you'll see the same programme all the time.